Library/Homebrew/.rubocop.yml

```diff
@@ -16,7 +16,7 @@ Lint/NestedMethodDefinition:
 Metrics/AbcSize:
   Max: 280
 Metrics/BlockLength:
-  Max: 106
+  Max: 111
```

Comment:
Which method is this needed for?

Reply:
This is for `install_args` which got a bit longer due to the `--debug-symbols` flag. This could also be handled by ignoring the rubocop check for that method only (since 111 is a pretty long method -- but in this case I followed the previous person's lead -- they had increased the length).

I didn't see how breaking up the method would increase usability at all (it's basically a DSL).

Reply:
yeh, I'd rather we had an ignore for that method (or, ideally, `*_args` methods) specifically 👍🏻

Reply:
What are you thinking when you say `*_args` methods? A method for each switch/flag? And then those all aggregated in the single method?

For now will do ignore (however).

Reply:
Ignore for file is done.